exercises/pangram/test/test_pangram.c

```diff
@@ -9,31 +9,25 @@ void tearDown(void)
 {
 }
 
-void test_null(void)
```

Comment:
There's a special case for this test in the example. Can we remove that special case now that this test is not included?

Reply:
I was in doubt between removing and keeping this test case. Although it is not explicitly in the canonical data, the majority of the exercises that I solved have tests to check null input, and it may be relevant to remember.

Reply:
I think there was a decision on the problem-specifications repo that null tests were unnecessary... I'll see if I can find the convo before I'll say for sure though

Reply:
Heres an issue that summarizes the current stance https://github.com/exercism/problem-specifications/issues/1071

Basically, we can check for invalid inputs such as `null` as an edge case argument for a string parameter in a per-track manner. But where languages don't allow for it, attempting to input integer arguments in for a string parameter would not make sense in languages that don't facilitate this kind of thing.

In this case that means, IMO, that it is basically up to the track maintainers if invalid cases should be tested. For my two c, if there is a test, and a check for that case in the implementation as-is, then leave it in.

Reply:
Thanks for the clarification. I added back the null test.